src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const uniqueRobotValues = [...new Set(robotValues)];
+
+  if (robotValues.length !== uniqueRobotValues.length) {
+    return null;
+  }
```

Comment:
You could have checked for duplicates with  ` if (fixedRobot.hasOwnProperty(robot[key])) {
      return null;
    }` inside the loop for, it would be shorter. But you solution also works.

Reply:
> You could have checked for duplicates with ` if (fixedRobot.hasOwnProperty(robot[key])) { return null; }` inside the loop for, it would be shorter. But you solution also works.

You're right, this solution would be simpler! Thanks for pointing this out😄